apps/web/src/components/Posts/PostComposer.tsx

```diff
@@ -107,7 +107,7 @@ export default function PostComposer({ onBackClick, tokenRef }: Props) {
             <TextAreaWithCharCount
               currentCharCount={description.length}
               maxCharCount={DESCRIPTION_MAX_LENGTH}
-              placeholder={`Say something about ${inputPlaceholderTokenName}`}
+              placeholder={`Say something about "${inputPlaceholderTokenName}"`}
```

Comment:
super random but maybe we could play with more fun quotations: https://www.piliapp.com/symbol/quotation-mark/